### PR TITLE
cmake: updated FindODBC.cmake to search right sqlext.h on Windows

### PR DESCRIPTION
**Pull request by snikulov**
Updated cmake to search for proper ODBC on Windows platform.

### CODE REVIEW

File: cmake/modules/FindODBC.cmake [hunk 1]
```diff
@@ -17,8 +17,11 @@
 # ODBC_LIBRARY, where to find the ODBC driver manager library.
 
 set(ODBC_FOUND FALSE)
+include(CheckIncludeFiles)
+check_include_files("windows.h;sqlext.h" HAVE_SQLEXT_H)
 
-find_path(ODBC_INCLUDE_DIR sql.h
+find_path(ODBC_INCLUDE_DIR sqlext.h
+  /include
```
Comment (vadz) [inline cmake/modules/FindODBC.cmake]: Just curious, do any platforms/compilers really have any headers in `/include`? I think I've never encountered this...

**Comment by snikulov** [comment]
@vadz Yep, you're right. Just forget to remove it. Fixed.

**Comment by snikulov** [comment]
@vadz looks like I break something on Linux. Will investigate it, and update PR soon.

**Comment by snikulov** [comment]
@vadz Voila! I'm done. Now all green :)

**Comment by vadz** [comment]
Thanks!

**Pull request merged**